benchmarks/nemo/nemolite2d/psykal/Makefile

```diff
@@ -53,44 +54,48 @@ all: nemolite2d_serial
 KERNEL_DIR = ../kernels/fortran
 
 # Shorthand for invoking PSyclone
-PSYCLONE = psyclone -api gocean1.0 -d ${KERNEL_DIR} -l
+PSYCLONE = psyclone -api gocean1.0 -d ${KERNEL_DIR} -l output
 
 # Serial PSyclone code-generation of NemoLite2D
-nemolite2d_serial: common_lib timer_lib serial_inf_lib nemolite2d_alg.f90
+nemolite2d_serial: serial_libs nemolite2d_alg.f90 psyclone_scripts/serial_transform.py
 	@echo "Generating NemoLite2D Serial"; rm -rf $@; mkdir -p $@
-	${PSYCLONE} -nodm -oalg $@/alg.f90 -opsy $@/psy.f90 nemolite2d_alg.f90
+	${PSYCLONE} -nodm -s psyclone_scripts/serial_transform.py \
+		-oalg $@/alg.f90 -opsy $@/psy.f90 nemolite2d_alg.f90
 	cp ${INF_LIB} $@/. # Each generated folder needs its purposely built inf_lib
 	cp namelist $@/.
 	cp Makefile_gen $@/Makefile
+	cd $@ && make nemolite2d
 	@echo "Serial version prepared."
-	@echo "Run 'make nemolite2d' inside the $@ folder."
 
 # OpenMP PSyclone code-generation of NemoLite2D
-nemolite2d_omp: common_lib timer_lib serial_inf_lib nemolite2d_alg.f90 omp_transform.py
+nemolite2d_omp: serial_libs nemolite2d_alg.f90 psyclone_scripts/omp_transform.py
 	@echo "Generating NemoLite2D with OpenMP using omp_transform.py"; rm -rf $@; mkdir -p $@
-	${PSYCLONE} -nodm -s ./omp_transform.py -oalg $@/alg.f90 -opsy $@/psy.f90 nemolite2d_alg.f90
+	${PSYCLONE} -nodm -s psyclone_scripts/omp_transform.py \
+		-oalg $@/alg.f90 -opsy $@/psy.f90 nemolite2d_alg.f90
 	cp ${INF_LIB} $@/. # Each generated folder needs its purposely built inf_lib
 	cp namelist $@/.
 	cp Makefile_gen $@/Makefile
+	cd $@ && make nemolite2d
 	@echo "OpenMP version prepared."
-	@echo "Run 'make nemolite2d' inside the $@ folder."
 
 
 # MPI PSyclone code-generation of NemoLite2D
-nemolite2d_mpi: common_lib timer_lib mpi_inf_lib nemolite2d_alg.f90
+nemolite2d_mpi: parallel_libs nemolite2d_alg.f90 psyclone_scripts/serial_transform.py
 	@echo "Generating NemoLite2D with MPI"; rm -rf $@; mkdir -p $@
-	${PSYCLONE} -dm -oalg $@/alg.f90 -opsy $@/psy.f90 nemolite2d_alg.f90
+	${PSYCLONE} -dm -s psyclone_scripts/serial_transform.py \
+	   	-oalg $@/alg.f90 -opsy $@/psy.f90 nemolite2d_alg.f90
 	cp ${INF_LIB} $@/. # Each generated folder needs its purposely built inf_lib
 	cp namelist $@/.
 	cp Makefile_gen $@/Makefile
+	cd $@ && make nemolite2d
 	@echo "MPI version prepared."
-	@echo "Run 'make nemolite2d' inside the $@ folder."
 
 # Hybrid PSyclone code-generation of NemoLite2D
-nemolite2d_hybrid: common_lib timer_lib mpi_inf_lib nemolite2d_alg.f90 omp_transform.py
+nemolite2d_hybrid: parallel_libs nemolite2d_alg.f90 psyclone_scripts/omp_transform.py
```

Comment:
The PSyclone step in this target fails for me:

    Transformation Error: Error in GOceanOMPLoopTrans transformation. The node is not a loop.

I'm using HEAD of master. Perhaps we should have PSyclone as submodule of PSycloneBench so that we can ensure that a working version is always available?

Reply:
I forgot the updated script in ScafellPike and was not part of the PR. It should work now.